### PR TITLE
prt_double fix

### PR DESCRIPTION
**Pull request by alexch-m**

### CODE REVIEW

**Comment by astand** [comment]
@alexch-m Hi, please fix merge conflicts and update README as well

File: src/helpers/formatter.cpp [hunk 1]
```diff
@@ -1,6 +1,9 @@
 #include "formatter.h"
 #include <algorithm>
```
Comment (astand) [inline src/helpers/formatter.cpp]: remove extra empty line

File: src/helpers/formatter.cpp [hunk 2]
```diff
@@ -144,59 +147,46 @@ std::string make_c_name(const std::string& s)
   return ret;
 }
 
-const char* prt_double(double value, size_t presicsion, bool usedot)
+std::string prt_double(double value, size_t precision, bool usedot)
```
Comment (astand) [inline src/helpers/formatter.cpp:149]: Run style script please

File: src/helpers/formatter.h [hunk 3]
```diff
@@ -21,8 +21,8 @@ std::string str_trim(std::string s);
 /// @param value value to format
 /// @param presicsion maximal precision length
```
Comment (astand) [inline src/helpers/formatter.h]: also fix precision in doxy

File: src/helpers/formatter.cpp (hunk 1 above)
Comment (alexch-m) [reply src/helpers/formatter.cpp]: done

File: src/helpers/formatter.cpp (hunk 2 above)
Comment (alexch-m) [reply src/helpers/formatter.cpp:149]: done

File: src/helpers/formatter.h (hunk 3 above)
Comment (alexch-m) [reply src/helpers/formatter.h]: done

**Review by astand — APPROVED** [review]

**Comment by astand** [comment]
Hi, @alexch-m , please rebase to the latest commit and update README also

**Comment by alexch-m** [comment]
> Hi, @alexch-m , please rebase to the latest commit and update README also

Regarding README, I don't see version info or alike in it, what do you mean?

**Comment by astand** [comment]
@alexch-m Yeah, sorry for confusion - it is docs/RELEASES.md

**Comment by alexch-m** [comment]
README updated

**Review by astand — APPROVED** [review]

**Pull request merged**